Add April-2022-astronomy-minutes.md (#3)
datacarpentry/curriculum-advisors: astronomy/April-2022-astronomy-minutes.md

# April 2022 Meeting of the Astronomy Curriculum Advisors  

There were initially two meetings arranged. But at the end, only one meeting took place on **March 25, 2022, 16.00-17.00 CET**.

## Attendants

* Azalee Bostroem
* Heather Andrews
* Ivelina Momcheva
* Phil Rosenfield
* Adam Hughes
* Rachel Lee McClure
* Erin Becker

## Roles

**Facilitator:** Rachel

**Notetaker:** Phil

**Timekeeper:** Erin

## Agenda

### Introductions

7 very interesting and excellent people!

### Discussion on transition of curriculum to Stable

* Curriculum has gone through the following phases: **dev**, **alpha**, **beta**. It has been taught several times, now (proposed) ready to transition to stable.

* **First time a CAC is making this decision** (rather than core carpentries team).

* **First new curriculum since 2018**. 

*  **Can this be taught over 2 days?**
	- Timing was set by The Carpentries.
	- All testing was done virtually (pandemic times), there are notes on what to cut for time or other reasons.
	- Still need more data points as we transition to in-person and more people teach it.

*  **What is the recommended level of participants?**

> 	- It's not immediately clear that this curriculum is intermediate / advanced. It has prerequisites (it is "level 2").

Reviewer: ```suggestion
	- This is the first DC curriculum that has prerequisites (it is "level 2"). We want to be sure our audience is aware of this.
	- prerequisites are listed at top of lesson
	- this is something we will be sure to include in advertising
	- there will soon be a second lesson with prerequisites - so this will be less unusual
```

*  **Any major gaps (beyond the issues raised by previous workshop feedback)?**
	- No major gaps.
	- (Azalee) We had funding to hire a curriculum developer to do the major lift; since then it's been moving topics around, simplifying whether to write a function or not. Most current issues will be closed by current pull requests; others are flagged by what needs to be done before release.

> 	- (Azalee) Worried that we don't have any data points on in-person. But don't think it's important to keep this in beta until we go in-person. Don't think anything will be a big enough change. [Consensus agreement]

Reviewer: ```suggestion
	-  Worried that we don't have any data points on in-person. But don't think it's important to keep this in beta until we go in-person. Don't think anything will be a big enough change. [Consensus agreement]
```

Reviewer: this was actually a comment by Phil but I don't think its important who said it

> *  **Other comments?**

Reviewer: I included this in the discussion above - I think it makes sense to include it in context

	- Image analysis w/Python will be another 2nd level curriculum.
	- Erin + team to manage community expectations.

### Voting on transition of curriculum to Stable

 All (6) present vote yes to move to stable.

### Preferences for asynchronous communication

We have 2 meetings, we meet quarterly. **Should we have a space for async communication?**

**Proposed:** Slack channel in The Carpentries workspace. 
	- No current member needs to be owner (when someone cycles off, the channel persists).
	- Consensus among this group to move forward with this proposal.
  
### Goals for the CAC for the year

* Mini lessons
	- E.g., AAS WWT pull in Hubble data and overlay the plot
	- Need maintenance plan, need communication to instructors on what is "extra"
	- Could happen on an institutional level so you could trade out the data and use case.

> * Could do other data as a full new lesson 

Reviewer: ```suggestion
    - Could do other data as a full new lesson 
```

Reviewer: this was part of the mini lesson discussion

> * Engage with the Carpentries incubator? The Carpentries Incubator is a place to share community developed lessons that are not curricula. 

Reviewer: ```suggestion
    - Engage with the Carpentries incubator? The Carpentries Incubator is a place to share community developed lessons that are not curricula. 
```

Reviewer: this was part of the mini lesson discussion
